src/test/java/core/basesyntax/MatrixOperationTest.java

```diff
@@ -1,8 +1,44 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorldTest {
+import org.junit.Assert;
+import org.junit.Test;
 
-}
\ No newline at end of file
+import java.util.Arrays;
+
+public class MatrixOperationTest {
+    private static final int[][] TEST_MATRIX = {
```

Comment:
@Sofasmile add at least one more example to avoid hardcoding the return result by students

Reply:
@Sofasmile you may use Parametrized test, but it is not required